low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -180,6 +180,23 @@ SYSTEMDWATCHDOG:
    monitor: true
    threaded: true
    monitored_services:
+      - hare-consul-agent.service
+      - hare-consul-agent-c1.service
+      - hare-consul-agent-c2.service
+      - elasticsearch.service
+      - kibana.service
```

Comment:
Kibana is managed by pacemaker. Please remove this.

Reply:
removed kibana service from list.